src/styles/partials/components/_button.scss

```diff
@@ -1,108 +1,112 @@
 .dg_button,
 .dg_button-secondary {
-	&:hover,
-	&.hovered {
-		background: $vibrant-blue;
-	}
-
-	&:active,
-	&.active {
-		background: $pale-blue;
-	}
-
-	&:disabled,
-	&.disabled {
-		border-color: $gray-light;
-	}
+  &:hover,
+  &.hovered {
+    background: $vibrant-blue;
```

Comment:
Taking in to account we'll be using these classes for `a` tags as well, I noticed when setting an existing `button` to an `a` tag, it also inherits the `color` property, originally associated with the default tag. This makes the `color` blend in with the background. I would recommend setting a `color` property here to compensate.

Reply:
@theilman7 can you show me this one?

Reply:
@martypowell Certainly can. I'll meet with you tomorrow morning. Circling back, this only appears to affect the Button Primary component... or more specifically `.dg_button`.

Reply:
After discussing with Marty, this will be a non-issue since we do not plan on using `.dg_button` as a link button.